src/components/learner-credit-management/cards/tests/CourseCard.test.jsx

```diff
@@ -548,7 +548,7 @@ describe('Course card works as expected', () => {
       expectedValidationMessage: 'The total assignment cost exceeds your available Learner Credit budget balance of $100. Please remove learners and try again.',
     },
     {
-      learnerEmails: ['a@a.com', 'b@b.com', 'c@c.com'],
+      learnerEmails: ['a@a.com', 'B@b.com', 'c@c.com'],
```

Comment:
[inform] Previously failing on `master` with the `B@b.com` email as described by the ticket.

Reply:
Can you add another test case including both `b@b.com` and `B@b.com`?  It's not the case we were testing during standup, but it's related enough IMO.

Reply:
I believe this existing test case just happened to be "passing" due this bug, not because it was a duplicate.

Reply:
I suppose if B@b.com appeared before b@b.com it would have failed!